sections/round_structure.tex

```diff
@@ -26,6 +26,13 @@
 
 \vspace*{\fill}
 
-\includegraphics[width=1.8\linewidth]{\art/minotaur.png}
+\includegraphics[width=1.6\linewidth]{\art/minotaur.png}
```

Comment:
Please move it slightly to the right.

Reply:
how?

Reply:
Use `\hspace`, see Medusa in Scenario End.

Reply:
Done